configuration/bind-dirs.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: doc
+title: Bind-dirs - make persisnent changes to arbitrary system file in AppVM
```

Comment:
How about: "How to make any file in a TemplateBasedVM persistent using bind-dirs"